docs/CHANGELOG.asciidoc

```diff
@@ -77,6 +78,353 @@ Review important information about the {kib} 8.x releases.
+* Allow `traces` to be added to the `monitoring_enabled` array in Agent policies ({kibana-pull}189908[#189908]).
+* Create task that periodically unenrolls inactive agents ({kibana-pull}189861[#189861]).
+* Adds setup technology selector to add integration page ({kibana-pull}189612[#189612]).
+* Support integration-level outputs ({kibana-pull}189125[#189125]).
+Discover::
+* Renames the Documents tab to Results in ES|QL mode ({kibana-pull}197833[#197833]).
+* Adds a cluster details tab for CCS data sources when inspecting requests in ES|QL mode ({kibana-pull}195373[#195373]).
+* Adds the query time to the list of statistics when inspecting requests in ES|QL mode ({kibana-pull}194806[#194806]).
+* Changes the auto-focus to be on the ES|QL editor when loading the page ({kibana-pull}193800[#193800]).
+* Updates the autocomplete behavior for `SORT` to be in line with other field-list-based experiences like `KEEP` in ES|QL queries ({kibana-pull}193595[#193595]).
+* Adds the ability to expand the height of various sections in the Edit ES|QL visualization flyout ({kibana-pull}193453[#193453]).
+* Adds `all (*)` to the list of suggestions for `COUNT` functions in ES|QL queries ({kibana-pull}192205[#192205]).
+* Enhances the inline documentation experience in ES|QL mode ({kibana-pull}192156[#192156]).
+* Improves ES|QL autocomplete suggestions for `case()` expressions ({kibana-pull}192135[#192135]).
+* Improves display of error messages in ES|QL mode ({kibana-pull}191320[#191320]).
+* Opens suggestions automatically for sources lists and `ENRICH` functions when writing ES|QL queries ({kibana-pull}191312[#191312]).
+* Improves wrapping and readability for ES|QL queries ({kibana-pull}191269[#191269]).
+* Improves suggestions based on previous function arguments and date suggestions for `bucket` functions in ES|QL queries ({kibana-pull}190828[#190828]).
+* Adds a help menu to the ES|QL mode ({kibana-pull}190579[#190579]).
+* Show the `LIMIT` information in the ES|QL editor's footer ({kibana-pull}190498[#190498]).
+* Opens suggestions automatically for field lists in ES|QL queries ({kibana-pull}190466[#190466]).
+* Adds support for client-side `INLINESTATS` validation for ES|QL queries ({kibana-pull}189827[#189827]).
+* Integrates a time picker for date fields into the ES|QL editor ({kibana-pull}187047[#187047]).
+* Improves the query authoring experience when editing an ES|QL visualization ({kibana-pull}186875[#186875]).
+* Deactivates client-side pagination for ES|QL results ({kibana-pull}189715[#189715]).
+* Improves ES|QL support for Elasticsearch sub-types in AST for both validation and autocomplete  ({kibana-pull}189689[#189689]).
+* Adds ECS information to the ES|QL editor suggestions and prioritizes fields based on ECS information on the editor ({kibana-pull}187922[#187922]).
+* Improves `BY` suggestions in ES|QL queries to include pipe and comma operators ({kibana-pull}189458[#189458]).
```

Comment:
These are ES|QL editor improvements. The editor exists in many places, not only in Discover. Can we possibly have an ES|QL editor section?